src/core/getters/params.ts

```diff
@@ -58,7 +58,7 @@ export const getParams = ({
         schema,
       } = pathParam.parameter;
 
-      const name = sanitize(camel(nameWithoutSanitize));
+      const name = sanitize(camel(nameWithoutSanitize), { es5keyword:true });
```

Comment:
will remove underscore and dash on the string if you use the sanitize function like this